lib/logstash/outputs/elasticsearch_java.rb

```diff
@@ -481,11 +481,7 @@ def flush(actions, teardown = false)
   end # def flush
 
   public
-  def teardown
-    if @cacert # remove temporary jks store created from the cacert
-      File.delete(@truststore)
-    end
-
```

Comment:
That was old code that should have been removed in a previous refactor.
Manticore handles that all now. And in fact in the Java plugin there is no
ssl support come to think of it !

On Thursday, September 24, 2015, Pier-Hugues Pellerin <
notifications@github.com> wrote:

> In lib/logstash/outputs/elasticsearch_java.rb
> <https://github.com/logstash-plugins/logstash-output-elasticsearch_java/pull/14#discussion_r40368219>
> :
>
> > @@ -481,11 +481,7 @@ def flush(actions, teardown = false)
> >    end # def flush
> >
> >    public
> > -  def teardown
> > -    if @cacert # remove temporary jks store created from the cacert
> > -      File.delete(@truststore)
> > -    end
> > -
>
> I am missing a bit of story here, we don't delete the temporary trustore?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/logstash-plugins/logstash-output-elasticsearch_java/pull/14/files#r40368219>
> .
>